pkg/workerpool/workerpool.go

```diff
@@ -0,0 +1,130 @@
+package workerpool
+
+import (
+	"runtime"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/openservicemesh/osm/pkg/logger"
+)
+
+const (
+	// Size of the job queue per worker
```

Comment:
A comment on how this number should be determined will help.

Reply:
I don't have measures for it yet. This happens to be roughly 8-to-10 times the current (expected) number of jobs we will have for what I believe will be its primary use case in short term, but this is complete ballparking.
As soon as we have integration changes, I plan to put in place some Prom counters to understand how the queues do under stress, which will later help determine better this number.